Dockerfiles/midstream/run_tests.py

```diff
@@ -53,32 +58,24 @@ def test_for_extract_and_validate_bundle_image(self):
         # check if IMAGE_TO_TEST is defined, return exit_code 102 in case it's not
         if (self.image_to_test is None):
             print("Environment variable IMAGE_TO_TEST not set! Stopping the tests.")
-            with open(".errormessage", 'w') as error_msg:
+            with open("/tmp/.errormessage", 'w') as error_msg:
                 print("Result code: 102 Error message: Environment variable IMAGE_TO_TEST not set!", file=error_msg)
             exit_code = 102
             return exit_code
-        if (self.operator_dir != '/project/operator_dir/' or
-           self.operator_work_dir != '/project/test_operator_work_dir/' or
-           self.playbooks_dir != '/project/operator-test-playbooks/' or
-           self.umoci_bin_path != '/usr/local/bin/umoci' or
-           self.work_dir != '/project/output/'):
-            print("Environment variable misconfigured!")
-            with open(".errormessage", 'w') as error_msg:
-                print("Result code: 103 Error message: Environment variable was not set correctly!", file=error_msg)
-            exit_code = 103
-            return exit_code
         result = self.run_extract_operator_bundle()
+        print(result.stdout)
         if (result.returncode != 0):
             print("Ansible playbook extract-operator-bundle.yml failed with result code: %s , see the file .errormessage for more info." % result.returncode)
-            with open(".errormessage", "w") as error_msg:
+            with open("/tmp/.errormessage", "w") as error_msg:
                 print("Result code: " + str(result.returncode), "Error message: " + str(result.stderr), file=error_msg)
             exit_code = 50
             return exit_code
         result = self.run_validate_operator_bundle()
-        assert (path.exists("/project/output/validation-rc.txt"))
-        assert (path.exists("/project/output/validation-output.txt"))
-        assert (path.exists("/project/output/validation-version.txt"))
-        with open('/project/output/validation-rc.txt', 'r') as reader:
+        print(result.stdout)
+        assert (path.exists(WORK_DIR+"/validation-rc.txt"))
+        assert (path.exists(WORK_DIR+"/validation-output.txt"))
+        assert (path.exists(WORK_DIR+"/validation-version.txt"))
```

Comment:
what is a goal of the assertions ?

Reply:
Goal of the assertions is to make sure the output files are created.

Reply:
But `validation-version.txt`and `validation-output.txt` are not used anywhere in the script.
Why do we check whether they exist ??

Reply:
If there are not created in the run. They remark that run was not successful. 
For every run these files need to be created. If not we should consider it as a failed run afaik.

Reply:
If we do not use that file than it does not have a reason to check it.